### DISCUSSION TITLE
Save multiple files from a loop

### DISCUSSION THREAD

**Discussion by N1coleR** [OpenDrift/opendrift#1201] [Q&A] [answered]
Hi everyone:

I am trying to release multiple "eggs" with a day of difference (start and end), the duration of the run for all the releases should be 5 days. I am having some trouble storing the data, the loop only store the first release, and in the second run I getting this error: 

AttributeError: 'PelagicEggDrift' object has no attribute 'environment'
During handling of the above exception, another exception occurred:
SystemExit: Simulation aborted. 'The simulation stopped before requested end time was reached.'

Has any of you solved something similar?
Below I leave the part of the code with problems

Thank you in advanced!!

model used: PelagicEggDrift

start_time = uv.start_time

for i in range(0,2): 
    o.seed_elements(Lon, Lat, time = start_time + timedelta(days=i))
    o.run(duration=timedelta(days=i+5), outfile='Proof'+str(i+1)+'.nc')

**Comment by AndresSepulveda** [comment, marked as answer]
Hi,


Just a suggestion

You can do a loop only for the seeding and set 

o.set_config('drift:max_age_seconds',612000) # 5*24*3600

to limit the drifting time of each particle. That way all particles can be stored in a single file.

**Reply by N1coleR** [reply]
Hi Andrés:
Thank you for the suggestion, in deed, it solve my problem but outside the loop. The idea was to release multiple "eggs" with a day of difference (1st relase: day 1 to 10; 2nd release: day 2 to 11, etc.)

start_time = uv.start_time

for i in range(0,2): 
    o.seed_elements(Lon, Lat, time = start_time + timedelta(days=i)) 
    
o.set_config('drift:max_age_seconds',864000) 
o.run(duration = timedelta(days = 11), outfile = 'proof3.nc')

--------------------------------------------------------------------------
I found other way to solve it, but I can't store the .NC, you can only store certain data (works for me too).

start_time = uv.start_time
Flon = [ ]
Flat = [ ]

for i in range(0,2): 
    o.seed_elements(Lon, Lat, time = start_time + timedelta(days=i)) 
    o.run(duration = timedelta(days = i + 5)) 
    FinLon = o.elements.lon
    FinLat = o.elements.lat
    Flon.append(FinLon)
    Flat.append(FinLat)
